model-optimizer/extensions/ops/elementwise.py

```diff
@@ -205,9 +206,17 @@ class Minimum(Elementwise):
 class Round(Elementwise):
     enabled = False
     op = 'Round'
-    op_type = None
-    version = 'extension'
+    op_type = 'Round'
+    version = 'opset4'
     operation = staticmethod(lambda a: np.round(a))
+    def __init__(self, graph: Graph, attrs):
```

Comment:
Please, create a base class `UnaryElementwise` and use it as a base class for `Round` op.

Reply:
Done. Also MO IR Reader updated to read `UnaryElementwise` operations